src/main/scala/sbtorgpolicies/settings/bash.scala

```diff
@@ -32,12 +36,14 @@ trait bash extends bashCompat {
     val st: State = deferredFetchContributorsState(inputState)
     val extracted = Project.extract(st)
 
-    val buildV    = extracted.get(version in ThisBuild)
-    val scalaV    = extracted.get(scalaVersion)
-    val crossV    = extracted.get(crossScalaVersions)
-    val orgBranch = extracted.get(orgCommitBranchSetting)
+    val buildV     = extracted.get(version in ThisBuild)
+    val scalaV     = extracted.get(scalaVersion)
+    val crossBuild = extracted.get(crossScalaVersions).toList
+    val orgBranch  = extracted.get(orgCommitBranchSetting)
+    val baseDir    = extracted.get(baseDirectory in LocalRootProject)
```

Comment:
Any reason this `baseDir` is different than https://github.com/47deg/sbt-org-policies/pull/709/files#diff-6325ccd2a085563a90de12681230e44aR157

Reply:
Oh, niche catch, I'll rename this to `rootDir` and use the `rootDir` [here](https://github.com/47deg/sbt-org-policies/pull/709/files#diff-6325ccd2a085563a90de12681230e44aR157)